Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md

```diff
@@ -0,0 +1,58 @@
+This playbook is designed to handle the following alerts:  
+- Possible LDAP enumeration by unsigned process (medium severity)
```

Comment:
let's keep only the alert name without the quotes. needs to be aligned with other playbooks for future automation processes and to make it easy for customer to adopt. 
You can say in a separate sentence or in the title if you want that those are severe alerts